### DISCUSSION TITLE
My academic status - error message

### DISCUSSION THREAD

**Discussion by LukasCerny** [community/community#68865] [GitHub Education]
### Hi everyone,

Dear Sir or Madam,

I would like to ask for your help in changing my academic status. I get the same error for every document (see photo), even though I have edited my GitHub profile, made 2FA and others. This is very frustrating because my university provides the Confirmation of Studies document in pdf format with an electronic signature. But as you know, pdf files are not supported for verification.

I even connected to university VDI (Virtual Desktop Infrastructure) to have the most closet position to our campus.

Could you please help me?

Thank you in advance.

Regards,
Lukas Cerny
![Error](https://github.com/community/community/assets/2876987/6f020fb7-0102-4dbe-a257-886be3f05153)

**Comment by harxh-v** [comment]
Dear @LukasCerny 

To address this issue, follow these steps:

1. **Update Your GitHub Profile**:
   - Log in to your GitHub account.
   - Go to your profile by clicking on your profile picture in the upper-right corner and selecting "Your profile."
   - Click the "Edit profile" button.

2. **Ensure Your Full Name Matches Academic Affiliation Document**:
   - In the "Name" section, ensure that your full name exactly matches the name on your academic affiliation document, including any middle names or initials.
   
3. **Add a Short Bio**:
   - Scroll down to the "Bio" section and enter a short bio about yourself.

4. **Save Changes**:
   - After making these updates, scroll to the bottom of the page and click the "Update profile" button to save your changes.

5. **Log Out and Log Back In**:
   - Log out of your GitHub account.
   - Log back in to ensure that the changes you made to your profile have taken effect.

6. **Reapply for Verification**:
   - Once you've updated your profile information and logged out and back in, you can try to reapply for verification using your academic email and student ID.

By following these steps and ensuring that your GitHub profile information aligns with your academic affiliation document, you should have a better chance of successfully verifying your education account on GitHub. If you continue to encounter issues, consider reaching out to GitHub support for further assistance, as they can provide more specific guidance based on your situation.
